code-search/code-search.ipynb

```diff
@@ -0,0 +1,609 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Code search with Qdrant\n",
+        "\n",
+        "This is a notebook demonstrating how to implement a code search mechanism using two different neural encoders - one general purpuse, and another trained specifically for code. Let's start with installing all the required dependencies."
+      ],
+      "metadata": {
+        "id": "FcZxcGxGRcoj"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Q95WbjsKW2KH"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install qdrant-client inflection sentence-transformers optimum onnx"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We are going to work with [Qdrant source code](https://github.com/qdrant/qdrant) that has been already converted into chunks. If you want to do it for a different project, please consider using one of the [LSP implementations](https://microsoft.github.io/language-server-protocol/) for your programming language. It should be fairly easy to build similar structures with the help of these tools."
+      ],
+      "metadata": {
+        "id": "35XOEgMTSHlk"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "iD8Fdu1SWw_7"
+      },
+      "outputs": [],
+      "source": [
+        "!wget https://storage.googleapis.com/tutorial-attachments/code-search/structures.jsonl"
```

Comment:
Would you like to use Huggingface Datasets instead? It's as programmatic as wget, with the advantage of us getting a sense of how many people download it?